clients/src/main/java/org/apache/kafka/common/requests/WriteTxnMarkersResponse.java

```diff
@@ -164,4 +164,8 @@ public static WriteTxnMarkersResponse parse(ByteBuffer buffer, short version) {
         return new WriteTxnMarkersResponse(ApiKeys.WRITE_TXN_MARKERS.parseResponse(version, buffer));
     }
 
+    @Override
+    public boolean shouldClientThrottle(short version) {
+        return version >= 1;
```

Comment:
It seems that this is an internal request and is never throttled.

Reply:
Thanks for catching this. It is fixed now.